### ISSUE TITLE
Too low default value for ReadTimeout and WriteTimeout for XmlRpcClient

### ISSUE THREAD

**Issue by qwazer** [bsorrentino/maven-confluence-plugin#256]
After upgrading to confluence-reporting-maven-plugin 7.3 from 6.7.2 it immediately fails with 

```
Caused by: org.apache.maven.plugin.MojoExecutionException: error generating report
    at org.bsc.mojo.AbstractBaseConfluenceMojo.execute (AbstractBaseConfluenceMojo.java:384)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.net.SocketTimeoutException: connect timed out
    at java.net.PlainSocketImpl.socketConnect (Native Method)
    at java.net.AbstractPlainSocketImpl.doConnect (AbstractPlainSocketImpl.java:399)
    at java.net.AbstractPlainSocketImpl.connectToAddress (AbstractPlainSocketImpl.java:242)
    at java.net.AbstractPlainSocketImpl.connect (AbstractPlainSocketImpl.java:224)
    at java.net.SocksSocketImpl.connect (SocksSocketImpl.java:392)
    at java.net.Socket.connect (Socket.java:609)
    at sun.security.ssl.SSLSocketImpl.connect (SSLSocketImpl.java:289)
    at sun.net.NetworkClient.doConnect (NetworkClient.java:177)
    at sun.net.www.http.HttpClient.openServer (HttpClient.java:474)
    at sun.net.www.http.HttpClient.openServer (HttpClient.java:569)
    at sun.net.www.protocol.https.HttpsClient.<init> (HttpsClient.java:265)
    at sun.net.www.protocol.https.HttpsClient.New (HttpsClient.java:372)
    at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient (AbstractDelegateHttpsURLConnection.java:203)
    at sun.net.www.protocol.http.HttpURLConnection.plainConnect0 (HttpURLConnection.java:1187)
    at sun.net.www.protocol.http.HttpURLConnection.plainConnect (HttpURLConnection.java:1081)
    at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect (AbstractDelegateHttpsURLConnection.java:189)
    at sun.net.www.protocol.http.HttpURLConnection.getOutputStream0 (HttpURLConnection.java:1367)
    at sun.net.www.protocol.http.HttpURLConnection.getOutputStream (HttpURLConnection.java:1342)
    at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream (HttpsURLConnectionImpl.java:246)
    at org.apache.xmlrpc.client.XmlRpcSunHttpTransport.writeRequest (XmlRpcSunHttpTransport.java:104)
    at org.apache.xmlrpc.client.XmlRpcStreamTransport.sendRequest (XmlRpcStreamTransport.java:151)
    at org.apache.xmlrpc.client.XmlRpcHttpTransport.sendRequest (XmlRpcHttpTransport.java:143)
    at org.apache.xmlrpc.client.XmlRpcSunHttpTransport.sendRequest (XmlRpcSunHttpTransport.java:69)
    at org.apache.xmlrpc.client.XmlRpcClientWorker.execute (XmlRpcClientWorker.java:56)
    at org.apache.xmlrpc.client.XmlRpcClient.execute (XmlRpcClient.java:167)
    at org.apache.xmlrpc.client.XmlRpcClient.execute (XmlRpcClient.java:137)
    at org.apache.xmlrpc.client.XmlRpcClient.execute (XmlRpcClient.java:126)
    at org.bsc.confluence.xmlrpc.Confluence.call (Confluence.java:901)
    at org.bsc.confluence.xmlrpc.Confluence.call (Confluence.java:867)
    at org.bsc.confluence.xmlrpc.Confluence.call (Confluence.java:853)
    at org.bsc.confluence.xmlrpc.Confluence.login (Confluence.java:146)
    at org.bsc.confluence.xmlrpc.XMLRPCConfluenceService.createInstanceDetectingVersion (XMLRPCConfluenceService.java:77)
    at org.bsc.confluence.ConfluenceServiceFactory$MixedConfluenceService.<init> (ConfluenceServiceFactory.java:39)
    at org.bsc.confluence.ConfluenceServiceFactory._tryCeateInstance (ConfluenceServiceFactory.java:183)
    at org.bsc.confluence.ConfluenceServiceFactory.createInstance (ConfluenceServiceFactory.java:217)
    at org.bsc.mojo.AbstractBaseConfluenceMojo.execute (AbstractBaseConfluenceMojo.java:369)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)

```

plugin uses SECONDS timeunit
```
        clientConfig.setConnectionTimeout( (int)ConfluenceService.getConnectTimeout(TimeUnit.SECONDS) );
        clientConfig.setReplyTimeout((int)Math.max( ConfluenceService.getReadTimeout(TimeUnit.SECONDS),
                                                    ConfluenceService.getWriteTimeout(TimeUnit.SECONDS)) );
```
but 
XmlRpcHttpRequestConfig uses MILLISECONDS timeunit
```
    /** Return the connection timeout in milliseconds
     * @return connection timeout in milliseconds or 0 if no set
     */
    int getConnectionTimeout();
    
    /** Return the reply timeout in milliseconds
     * @return reply timeout in milliseconds or 0 if no set
     */
    int getReplyTimeout();
```

**Comment by bsorrentino** [comment]
Hi @qwazer 

I've deployed version `7.3.1-SNAPSHOT` with fix

**Comment by bsorrentino** [comment]
Fix release in `7.3.1`
